lib/caching-browserify.js

```diff
@@ -95,7 +95,8 @@ module.exports = CoreObject.extend({
       packageCache: {},
       fullPaths: this.fullPaths,
       basedir: this._inputStaging,
-      debug: this.enableSourcemap
+      debug: this.enableSourcemap,
+      browserField: !process.env.EMBER_CLI_FASTBOOT
```

Comment:
`EMBER_CLI_FASTBOOT` is not a thing in fastboot 1.0, there is only 1 build now, now two (fastboot + non-fastboot)